client/lib/signup/test/step-actions.js

```diff
@@ -27,7 +27,6 @@ jest.mock( 'lib/user', () => {
 } );
 
 jest.mock( 'signup/config/steps', () => require( './mocks/signup/config/steps' ) );
-jest.mock( 'signup/config/steps-pure', () => require( './mocks/signup/config/steps-pure' ) );
```

Comment:
I'm not sure how this wasn't failing before, but this `steps-pure` mock does not exist.